profile2/tool/src/webapp/css/profile2.css

```diff
@@ -15,342 +15,327 @@
  */
```

Comment:
This file will be removed when the PR is moved out of draft mode. These changes have been moved into profil2.scss under library. Keeping them here makes the review easier to scan and understand.